### ISSUE TITLE
portals.conf(5) not installed

### ISSUE THREAD

**Issue by jtojnar** [NixOS/nixpkgs#269577]
https://search.nixos.org/options?channel=unstable&show=xdg.portal.config says I'm supposed to check `portals.conf(5)` but `apropos portals.conf` returns nothing. Is there something wrong with the manpage?

nixos-unstable

_Originally posted by @sagehane in https://github.com/NixOS/nixpkgs/issues/247766#issuecomment-1825120120_

**Comment by Alizter** [comment]
Man pages are compressed by default, could this be having an effect. You can check it exists by finding the store entry and looking for the man pages.

**Comment by sagehane** [comment]
Just checked and `/nix/store/hgynmw3z684rcfk1418kdll49py8qkij-xdg-desktop-portal-1.18.1/share/man/man5/portals.conf.5.gz` is a valid path on my setup but it doesn't seem to symlink into `/run/current-system/sw/share/man/man5`.

`MANPATH=/nix/store/hgynmw3z684rcfk1418kdll49py8qkij-xdg-desktop-portal-1.18.1/share/man man portals.conf` works fine.

**Comment by Alizter** [comment]
@sagehane When you did `nixos-rebuild` did you restart your machine? Sometimes that can have an effect.

**Comment by sagehane** [comment]
> @sagehane When you did `nixos-rebuild` did you restart your machine? Sometimes that can have an effect.

I've rebooted like 2-3 times while testing the new XDG config stuff. I've also rebuilt the system about 5 times too. Everything other than the manpage seems to work at a glance. Maybe the store is corrupted and that caused issues? I doubt it.

**Comment by Alizter** [comment]
I suppose if we can reproduce in a vm then it definitely is a bug with the gnome setup that needs to be looked into. I haven't been able to build a vm since it crashes on boot probably for some unrelated reason. I don't have any idea what needs to be fixed here unfortunately.

**Comment by jtojnar** [comment]
Yes, the issue is that the file is not installed (linked), we currently only link subset of files. #109060 might fix that.

**Comment by tomodachi94** [comment]
@jtojnar can you comment on whether #268197 fixes this bug for you?

**Comment by jtojnar** [comment]
Yeah, cannot reproduce any more – that was probably the fix.
